docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md

```diff
@@ -0,0 +1,128 @@
+<!--
+title: "Troubleshooting Agent with Cloud connection"
+description: "A simple guide to troubleshoot occurances where the Agent is showing as offline after claiming."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md
+-->
+
+# Troubleshooting Agent with Cloud connection
+
+Sometimes, when claiming a node, it might not show up as online in Netdata Cloud.  
+The occurances triggering this behavior might be:
+
+- [The claiming script failed](#the-claiming-script-failed)
+- [Claiming on an older, deprecated version of the Agent](#claiming-on-an-older-deprecated-version-of-the-agent)
+- [Network issues while connecting to the Cloud](#network-issues-while-connecting-to-the-cloud)
```

Comment:
An additional one we could add here is the case where agent couldn't be reached and needs a restart after install/claim

Reply:
in this case the claiming script doesn't necessarily say it failed. But in the logs there will be a statement that agent needs to be restarted, could be hard to spot for users. So good to call it out I think.

Reply:
Will replace "failed" with "unsuccessful", and then before pointing out "if the Agent is not running" put a tip that the solution might be to restart it